src/tests/ftest/nvme/nvme_pool_extend.py

```diff
@@ -0,0 +1,242 @@
+            API (str): IOR API
+            test (list): IOR test sequence
+            flags (str): IOR flags
+            results (queue): queue for returning thread results
+        """
+        processes = self.params.get("slots", "/run/ior/clientslots/*")
+        mpio_util = MpioUtils()
+        if mpio_util.mpich_installed(self.hostlist_clients) is False:
+            self.fail("Exiting Test: Mpich not installed")
+        self.pool = pool
+        # Define the arguments for the ior_runner_thread method
+        ior_cmd = IorCommand()
+        ior_cmd.get_params(self)
+        ior_cmd.set_daos_params(self.server_group, self.pool)
+        ior_cmd.dfs_oclass.update(oclass)
+        ior_cmd.api.update(api)
+        ior_cmd.transfer_size.update(test[0])
+        ior_cmd.block_size.update(test[1])
+        ior_cmd.flags.update(flags)
+        if "-w" in flags:
+            self.container_info["{}{}{}"
+                                .format(oclass,
+                                        api,
+                                        test[0])] = str(uuid.uuid4())
+
+        # Define the job manager for the IOR command
+        manager = Mpirun(ior_cmd, mpitype="mpich")
+        key = "".join([oclass, api, str(test[0])])
+        manager.job.dfs_cont.update(self.container_info[key])
+        env = ior_cmd.get_default_env(str(manager))
+        manager.assign_hosts(self.hostlist_clients, self.workdir, None)
+        manager.assign_processes(processes)
+        manager.assign_environment(env, True)
+
+        # run IOR Command
+        try:
+            manager.run()
+        except CommandFailure as _error:
+            results.put("FAIL")
+
+    def run_ior_thread(self, action, oclass, api, test):
+        """[summary]
+        Args:
+            action (str): Start the IOR thread with Read or
+                          Write
+            oclass (str): IOR object class
+            API (str): IOR API
+            test (list): IOR test sequence
+            flags (str): IOR flags
+        """
+        if action == "Write":
+            flags = self.ior_w_flags
+        else:
+            flags = self.ior_r_flags
+
+        # Add a thread for these IOR arguments
+        process = threading.Thread(target=self.ior_thread,
+                                   kwargs={"pool": self.pool,
+                                           "oclass": oclass,
+                                           "api": api,
+                                           "test": test,
+                                           "flags": flags,
+                                           "results":
+                                           self.out_queue})
+        # Launch the IOR thread
+        process.start()
+        # Wait for the thread to finish
+        process.join()
+
+    def run_nvme_pool_extend(self, num_pool):
+        """Run Pool Extend
+            Args:
+             int : total pools to create for testing purposes.
```

Comment:
```suggestion
           Args:
               int : total pools to create for testing purposes.
```
Indentations aren't consistent.

Reply:
Done.